apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -202,6 +202,51 @@ const loadListByChunks = async ({
 
     return all
 }
+/**
+ * A simple way to load any model by chunks and process the resulting chunk.
+ * After processing, the chunk will be unloaded from memory
+ * @param context
+ * @param list
+ * @param {Object} where
+ * @param {String[]} sortBy
+ * @param {Number} chunkSize
+ * @param {Number} limit
+ * @param {function(Array): Array} chunkProcessor A place to use or/and modify just loaded chunk
+ * @returns {Promise<void>}
+ */
+const processListByChunks = async ({
```

Comment:
why not use find?

Reply:
because `find` can't sort and you can't offset, it returns all data with all fields

Reply:
Check ADR-36